exporters/otlp/testing-internal/src/main/java/io/opentelemetry/exporter/otlp/testing/internal/AbstractGrpcTelemetryExporterTest.java

```diff
@@ -1131,6 +1135,130 @@ void stringRepresentation() throws IOException, CertificateEncodingException {
     }
   }
 
+  @Test
+  void latestInternalTelemetry() {
+    if ("profile".equals(type)) {
```

Comment:
#nit Idiomatic way to do this with assert j is: `assumeThat(type).isNotEqualTo("profile")`

Reply:
Nice, interesting that I haven't come across this before (or can't remember...)